patches/build-android-gyp-lint.py.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/build/android/gyp/lint.py b/build/android/gyp/lint.py
+index f3d85f78ec1298c4e57e5811a3faede44edbc5f0..9ffe11d7eb1da442817695f1470509c618e7fff9 100755
+--- a/build/android/gyp/lint.py
++++ b/build/android/gyp/lint.py
+@@ -540,6 +540,7 @@ def main():
+   if args.depfile:
+     action_helpers.write_depfile(args.depfile, args.stamp, depfile_deps)
+ 
++from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
```

Comment:
nit: pls put it riiight before `if __name__ == '__main__':` line.

```
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
if __name__ == '__main__':

Reply:
thanks, done https://github.com/brave/brave-core/pull/22026/commits/ec546a08c71b44fbcb08b4fd3a6ad7b89878d085